indico/modules/events/persons/client/js/EmailSurveyParticipantsButton.jsx

```diff
@@ -0,0 +1,45 @@
+// This file is part of Indico.
```

Comment:
These files do not really belong in the "persons" module. Put them in the "surveys" module.